addons/binding/org.openhab.binding.network.test/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I mean the file not the xml declaration, I think you miss a lot of dependencies / plugins.